src/django/api/views.py

```diff
@@ -686,6 +686,14 @@ def destroy(self, request, pk=None):
                             'promoted_oar_id': promoted_facility.id,
                         })
                         other_item.save()
+
+                for alias in FacilityAlias.objects.filter(facility=facility):
+                    alias.facility = promoted_facility
+                    alias.changeReason = 'Deleted {} and promoted {}'.format(
+                        facility.id,
+                        promoted_facility.id
+                    )
+                    alias.save()
```

Comment:
Updating a `FacilityAlias` in place ends up crashing the Django Admin because django-simple-history tries to reference a now deleted object.

Reply:
Fixed in 1c25bc0